packages/cli/src/constructs/check.ts

```diff
@@ -197,6 +322,11 @@ export abstract class Check extends Construct {
     return config
   }
 
+  /**
+   * Creates alert channel subscriptions for this check.
+   * Links the check to its configured alert channels so notifications are sent.
+   * Only creates subscriptions if alert channels are configured and check is not test-only.
+   */
   addSubscriptions () {
```

Comment:
This should be a `protected` method tbh - it's not mean to be called by users

Reply:
Let's keep this as is for now to avoid any potentially breaking changes in the PR, but might want to rethink in the next major release.